packages/client/lib/peers.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * List from https://github.com/ArkEcosystem/peers
+ */
+module.exports = {
+  devnet: [
```

Comment:
We should avoid hardcoding peers as they will be modified by tharude or adrian here https://github.com/ArkEcosystem/peers and then you have to copy over the lists again from there.

Reply:
i think we should have both: in case of unavailability of github the whole network could be rekt

Reply:
Maybe we could try to use those harcoded peers first, and, in case all of them failed, use GH as a fallback.

Reply:
Other option, maybe better than requesting to GH, is creating a new NPM package (`@arkecosystem/peers`) on the [peers repository](https://github.com/ArkEcosystem/peers). That way, we'd be sure that the peers are updated.

Reply:
Yes, why don't you guys use the peers repo (https://github.com/ArkEcosystem/peers) instead of hard coding them? This way you'll have to update the peers only on one place.

Reply:
Any more thoughts on this Juan?